docs/dev/connectors/kinesis.md

```diff
@@ -129,6 +129,181 @@ shard IDs are not consecutive (as result of dynamic re-sharding in Kinesis).
 For cases where skew in the assignment leads to significant imbalanced consumption,
 a custom implementation of `KinesisShardAssigner` can be set on the consumer.
 
+### Using Enhanced Fan Out
```

Comment:
I'm not sure if it makes sense to place EFO at the top of the "advanced usage sections". It seems that this topic is not relevant for first-time users, rather for users later in the adoption processes.
Maybe it would make sense to move it past "Fault Tolerance for Exactly-Once User-Defined State Update Semantics", so that first-time users have a better experience.

Reply:
Moved as per suggestion